cmd/zpool/zpool_main.c

```diff
@@ -972,7 +972,7 @@ zpool_do_remove(int argc, char **argv)
 	int i, ret = 0;
 	zpool_handle_t *zhp = NULL;
 	boolean_t stop = B_FALSE;
-	char c;
+	int c;
```

Comment:
it can be changed to `signed char` in this case

Reply:
I think better to use an `int`, because `getopt()` is defined to return an `int`, and for consistency with all the other `getopt()` calls.